R/check_meta_label.R

```diff
@@ -0,0 +1,59 @@
+#' Check every row has a label
+#'
+#' Ensure there are no blank cells in the label column for every row in the metafile
+#'
+#' @inheritParams precheck_meta_col_type
+#'
+#' @inherit check_filename_spaces_return
```

Comment:
```suggestion
#' @inherit check_filename_spaces
```

This needs to be the exact function name - I think you need to drop the `_return` from the end of this

Reply:
Dropped the underscore before return so reads as #' @inherit check_filename_spaces return, consistent with other checks